pkg/sourcereader/embedded_test.go

```diff
@@ -156,3 +156,15 @@ func (s *MySuite) TestGetModule_Embedded(c *C) {
 	expectedErr = "Source is not valid: .*"
 	c.Assert(err, ErrorMatches, expectedErr)
 }
+
+func (s *MySuite) TestGetModule_NilFs(c *C) {
+	ModuleFS = nil
+	r := EmbeddedSourceReader{}
+	c.Assert(r.GetModule("here", "there"), ErrorMatches, "embedded file system is not initialized")
```

Comment:
I think it is generally bad golang practice to directly test for error string. I will share links for alternatives.

Reply:
Acknowledged. Discussed offline.